paths_cli/commands/contents.py

```diff
@@ -22,16 +22,16 @@
 
 @click.command(
     'contents',
-    short_help="list named objects from an OPS .nc file",
+    short_help="list named objects from an OPS output file",
```

Comment:
wouldn't this more suitable to be `storage` file, or just `file`?
```suggestion
    short_help="list named objects from an OPS storage file",
```
```suggestion
    short_help="list named objects from an OPS file",
```